### PR TITLE
Use nLog 4.5.0-rc06

### PR DESCRIPTION
**Pull request by AnthonySteele**
Use nLog 4.5.0-rc06
and update test tools

### CODE REVIEW

File: src/NLog.StructuredLogging.Json.Tests/NLog.StructuredLogging.Json.Tests.csproj [hunk 1]
```diff
@@ -19,8 +19,8 @@
     </Content>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="FakeItEasy" Version="4.4.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.5.0" />
+    <PackageReference Include="FakeItEasy" Version="4.5.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.6.0" />
```
Comment (martincostello) [inline src/NLog.StructuredLogging.Json.Tests/NLog.StructuredLogging.Json.Tests.csproj:23]: Interesting - this must've only come out in the last few days. I wonder what it adds?

**Review by martincostello — APPROVED** [review]

File: src/NLog.StructuredLogging.Json.Tests/NLog.StructuredLogging.Json.Tests.csproj (hunk 1 above)
Comment (AnthonySteele) [reply src/NLog.StructuredLogging.Json.Tests/NLog.StructuredLogging.Json.Tests.csproj:23]: I was just looking. Yes it's new this week.
It doesn't seem like it changes much that we use: https://github.com/Microsoft/vstest-docs/blob/master/docs/releases.md#1560

Comment (martincostello) [reply src/NLog.StructuredLogging.Json.Tests/NLog.StructuredLogging.Json.Tests.csproj:23]: Looks like various small reliability fixes and improvements for MSTest compared to `15.5.0`. Doesn't look like enough for me to bother making an update just to it in applications. I'll just update it in other things next time I need to change them.

Comment (AnthonySteele) [reply src/NLog.StructuredLogging.Json.Tests/NLog.StructuredLogging.Json.Tests.csproj:23]: Sure, though this is only used in NLog.StructuredLogging.Json 's test suite.

**Pull request merged**